build/yamls/externalnode/conf/antrea-agent.conf

```diff
@@ -32,6 +32,18 @@ featureGates:
 # Defaults to "k8sNode". Valid values include "k8sNode", and "externalNode".
 nodeType: externalNode
 
+externalNode:
+  # The expected Namespace in which the ExternalNode is created.
+  # Defaults to "default".
+  #externalNodeNamespace: default
+
+  # The policyBypassRules describes the traffic that is expected to bypass NetworkPolicy rules.
+  # Each rule contains the following four attributes:
+  # direction (ingress|egress), protocol(tcp/udp/icmp/ip), remote IP or CIDR, dst port (ICMP doesn't require),
+  # and these attributes are separated by ":", e.g. ["ingress:tcp:1.1.1.1:22", "egress:tcp:2.2.2.2:80"].
```

Comment:
this is a YAML file, so why don't we use more structured data?
```yaml
policyBypassRules:
    - direction: ingress
      protocol: tcp
      cidr: 1.1.1.1/32
      port: 22
```

Reply:
This code is from this PR https://github.com/antrea-io/antrea/pull/3799 @mengdie-song would you address it or share your thoughts?

Reply:
Thanks for the suggestion. I will update this part in my change.